lang/po/Firmware_cs.po

```diff
@@ -1422,13 +1445,13 @@ msgstr ""
 #. MSG_TITLE_PULLEY_CANNOT_MOVE c=20
 #: ../../Firmware/mmu2/errors_list.h:122 ../../Firmware/mmu2/errors_list.h:160
 msgid "PULLEY CANNOT MOVE"
-msgstr ""
+msgstr "KLADKA SE NEOTACI"
 
 #. MSG_PROGRESS_PARK_SELECTOR c=20
 #: ../../Firmware/mmu2_progress_converter.cpp:26
 #: ../../Firmware/mmu2_progress_converter.cpp:57
 msgid "Parking selector"
-msgstr ""
+msgstr "KLADKA SE NEOTACI"
```

Comment:
:thinking: how did this happen? In the translation tool there is "Parkovani selektoru"

Reply:
Copy paste issue on my side maybe.